src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -109,13 +109,29 @@ if( ${LSMCHOICE} > 1 ) then # CatchCN Only
     ln -s /discover/nobackup/projects/gmao/ssd/land/l_data/LandBCs_files_for_mkCatchParam/V001/FPAR_CDF_Params-M09.nc4
 endif
 
-########################################################################
-#    if $LADAS_COUPLING == 1 use $FVWORK and copy forcing to Scratch
-########################################################################
+#######################################################################
+# if $LADAS_COUPLING == 1:  LDAS coupled to central ADAS simulation
+#######################################################################
 
 if ( $LADAS_COUPLING == 1 ) then
-   echo "copying lfo_Nx+- met forcing from $FVWORK to $SCRDIR"
-   /bin/cp -f $FVWORK/*lfo_Nx+-*nc4  $SCRDIR/.
+
+   if ( $ENSEMBLE_FORCING == "YES" ) then
+
+      # create perturbed forcing from central simulation and atm ensemble
+
+      module load python/GEOSpyD/Ana2019.03_py3.7
```

Comment:
@saraqzhang: Why do we need to load python here?  I think GMAO_env (currently v4.2.0) loads  python/GEOSpyD/Min4.11.0_py3.9_AND_Min4.8.3_py2.7:
https://github.com/GEOS-ESM/ESMA_env/blob/b4176aa8c4639fa9f3dac7db1a0f061c30225930/g5_modules#L131
Could there be a conflict?

Reply:
I used 3.7 in the offline development and test.  I can test it using 2.7  see if any problem.

Reply:
There's also "py3.9" in the module that comes with GMAO_env.  I don't understand whether the GMAO_env module is v2 or v3 of python.  Maybe check with Matt?

Reply:
Addressed by @saraqzhang on 28 Jul, see comment below.  Marking conversation as "resolved".